android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -1346,7 +1346,7 @@ public static Typeface getFontTypeface(Context context, String fontFilename) {
         }
 
         File file = new File(fontFilename);
-        if (file.exists()) {
+        if (file != null && file.exists() && file.length() > 0) {
           return Typeface.createFromFile(file);
         }
```

Comment:
Can we put a try/catch around Typeface.createFromFile instead of just testing file length, which is just one of many possible corruption scenarios?

Reply:
This is currently in a try catch block (caught on l.1355).

https://github.com/keymanapp/keyman/blob/22b7380a5b1956a2720edc1765fe07c60e9a85d1/android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java#L1353-L1355

Do you want the catch to silently return null? the KM.LogException is what's generating the Toast notification

Reply:
It's one of those things that's really difficult. It's good if it notifies us without worrying the user, so can we silently log the exception?

Reply:
Ah, I had to review KMLog.LogException().

We only generate Toast notifications on non-stable builds. So then maybe we abandon this and leave things as is?

Reply:
Seems like a plan!